Sources/StreamChatSwiftUI/ChatChannel/ChatChannelViewModel.swift

```diff
@@ -398,12 +401,12 @@ import SwiftUI
     
     func dataSource(
         channelDataSource: ChannelDataSource,
-        didUpdateChannel channel: EntityChange<ChatChannel>
+        didUpdateChannel channel: ChatChannel
```

Comment:
I ended up changing the code to use the received value and not accessing `self.channel` which would still point at the previous value.